models/Recipe.model.js

```diff
@@ -2,7 +2,38 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title : {type:String,required:true},
+  level :{
+          type:String,
+          required:true , 
+          enum:['Easy Peasy',"Amateur Chef",'UltraPro Chef']
+        
+        },
+  ingredients : [String],//doubt
+  // ingredients: [
+  //   {
+  //     type: String
+  //   }
+  // ]
+  cuisine : String,
+  // cuisine: {
+  //   type: String
+  // }
+           
+  dishType : {
+    type:String,
+    enum:['breakfast','main_course','soup','snack', 'drink', 'dessert', 'other']
+  },
+  image: {type:String,
```

Comment:
check code cleanness here, click enter after the "{" to start a new line for multiline code